### PR TITLE
Initial docsvcs setup

### PR DESCRIPTION
**Pull request by dornera**

### CODE REVIEW

**Comment by dornera** [comment]
This is just the first take of setting up the pieces for docsvcs. Will most likely need to modify as we work to bring this towards production.

File: apps/docsvcs/docsvcs.tf [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  subnet_ids                  = ["${module.shared.private_subnets}"]
+  security_group_ids          = ["${module.eb_docsvcs.security_group_id}"]
+  major_engine_version        = "5.5"
+  allow_major_version_upgrade = "false"
+  apply_immediately           = "true"
+  dns_zone_id                 = "${module.shared.private_zoneid}"
+}
+
+module "eb_docsvcs" {
+  source = "../../../tf-mod-elasticbeanstalk-env"
```
Comment (gravesm) [inline apps/docsvcs/docsvcs.tf]: I don't think this is what we want?

Comment (dornera) [reply apps/docsvcs/docsvcs.tf]: You don't want to run locally from my computer? :)

Comment (dornera) [reply apps/docsvcs/docsvcs.tf]: All set

**Pull request merged**